src/App.js

```diff
@@ -185,7 +396,12 @@ export default function App() {
   }
 
   function resetStates() {
-    setSequenceCards([])
+    setSequenceCards({
+      '1st': [],
+      '2nd': [],
+      interview: [],
+      special: [],
+    })
     setUpdateIndex(null)
     setUpdateCard('')
   }
```

Comment:
You should definitely reduce the size of you app.js, extract some functions to other files and stuff.